src/main/resources/config/application-prod.yml

```diff
@@ -44,16 +50,21 @@ spring:
         prepStmtCacheSize: 250
         prepStmtCacheSqlLimit: 2048
         useServerPrepStmts: true
+        socketFactory: com.google.cloud.sql.mysql.SocketFactory
+        cloudSqlInstance: $SECRET_VALUE
+      jdbcUrl: $SECRET_VALUE
+      username: $SECRET_VALUE
+      password: $SECRET_VALUE
   jpa:
     show-sql: false
   # Replace by 'prod, faker' to add the faker context and have sample data loaded in production
   liquibase:
     contexts: prod
   mail:
-    host: localhost
+    host: $SECRET_VALUE
```

Comment:
any specific reason to use `$SECRET_VALUE` in all places? I prefer to revert unless they serve a specific purpose

Reply:
Good point. I thought about it for a moment and put it there just so that we could keep track of which values are being replaced. However yes it can be removed. :smile:

Reply:
Done. :+1: